packages/frontend/src/components/staking/components/Validator.js

```diff
@@ -57,6 +101,7 @@ export default function Validator({
                 <Translate id='staking.validator.button' />
             </FormButton>
             {validator && <StakingFee fee={validator.fee.percentage} />}
+            {validator?.version === PROJECT_VALIDATOR_VERSION && <StakingAPY apy={calculatedAPY} />}
```

Comment:
maybe we should make version not optional

Reply:
It has to be optional in this case, as validator might be undefined on page load